src/main/webapp/dev/js/main/instructorCourseEnrollPage.js

```diff
@@ -159,17 +158,15 @@ function expandCollapseExistingStudentsPanel() {
 function expandCollapseNewStudentsPanel() {
     const $panelHeading = $(this);
     const panelCollapse = $panelHeading.parent().children('.panel-collapse');
-    const displayIcon = $panelHeading.children('.display-icon');
+    const displayStatus = $panelHeading.children('.display-icon');
     const toggleChevron = $panelHeading.parent().find('.glyphicon-chevron-down, .glyphicon-chevron-up');
 
-    toggleStudentsPanel($panelHeading, panelCollapse, displayIcon, toggleChevron);
-    enrollHandsontable.render();
-
-    if ($(panelCollapse[0]).hasClass('checked')) { // if panel is shown
+    if (toggleStudentsPanel($panelHeading, panelCollapse, displayStatus, toggleChevron)) { // if panel is expanded
         $('.enroll-students-spreadsheet-buttons').show();
     } else {
         $('.enroll-students-spreadsheet-buttons').hide();
     }
+    enrollHandsontable.render();
```

Comment:
This last part looks like `adjustStudentsPanel` except for the buttons. By the way, you can use `$(...).toggle()` instead of detecting current state and use `.show()` or `.hide()` accordingly